Doc/library/urllib.parse.rst

```diff
@@ -575,6 +575,16 @@ task isn't already covered by the URL parsing functions above.
    Example: ``unquote_to_bytes('a%26%EF')`` yields ``b'a&\xef'``.
 
 
+.. function:: unquote_to_bytes_plus(string)
+
+   Like :func:`unquote_to_bytes`, but also replace plus signs by spaces, as required for
+   unquoting HTML form values.
+
+   *string* must be a :class:`str`.
+
+   Example: ``unquote_to_bytes_plus('/El+Ni%C3%B1o/')`` yields ``b'/El Niño/'``.
```

Comment:
Python 3 bytes strings only allow ASCII literals

Reply:
well spotted! I've fixed the example.